config/config.go

```diff
@@ -134,6 +134,19 @@ func (config *MixConfig) CreateDefaultConfig(localrpms bool) error {
 	return config.SaveConfig()
 }
 
+// CreateCustomConfig creates a default builder.conf using the pathPrefix
+// directory as base path for the variables values.
+func (config *MixConfig) CreateCustomConfig(localrpms bool, pathPrefix string) error {
```

Comment:
This is completely unused? I expect you intended to call it somewhere or expose it via cmdline.

Reply:
It's used in the test instead of percolating the changes all over the code base to support a pathPrefix.

Reply:
You resolved this comment but didn't respond to my suggestion.

Reply:
removed it :)